geonode/base/api/views.py

```diff
@@ -1497,9 +1497,9 @@ def linked_resources(self, request, pk):
                 # to get the resourcebase objects
                 resources = ResourceBase.objects.filter(pk__in=resources.values_list("object_id", flat=True))
 
-            additional_query_params = request.query_params
-            if additional_query_params:
-                resources = resources.filter(**{x: y for x, y in request.query_params.items()})
+            _filters = {x: y for x, y in request.query_params.items() if x not in ["page_size", "page"]}
```

Comment:
Question: Is it possible that `request.query_params` is `None`? In that case the code should handle such situation.

Reply:
In case django return an empty dictionary, but I'll improve the check to about that case